team-b-batch/src/main/kotlin/com/address/dsmproject/service/SaveAddressService.kt

```diff
@@ -0,0 +1,73 @@
+package com.address.dsmproject.service
+
+import com.address.dsmproject.feign.AddressClient
+import com.address.dsmproject.scheduler.AddressUtil.ENG_REAL_FILE_NAME
+import com.address.dsmproject.scheduler.AddressUtil.ENG_REQ_TYPE
+import com.address.dsmproject.scheduler.AddressUtil.ENG_ZIP_NAME
+import com.address.dsmproject.scheduler.AddressUtil.KOR_REAL_FILE_NAME
+import com.address.dsmproject.scheduler.AddressUtil.KOR_REQ_TYPE
+import com.address.dsmproject.scheduler.AddressUtil.KOR_ZIP_NAME
+import com.address.dsmproject.scheduler.AddressUtil.ZIP
+import org.springframework.stereotype.Service
+import java.io.File
+import java.nio.file.Files
+import java.nio.file.Paths
+import java.time.LocalDateTime
+import java.util.zip.ZipFile
+
+@Service
+class SaveAddressService(
+    private val addressClient: AddressClient,
+) {
+
+    fun execute(reqType: String, zipFilePath: String, filePath: String) {
+        Files.write(Paths.get(zipFilePath), getAddressInfo(reqType))
+        val zipFile = File(zipFilePath)
+        val unzipFile = File(filePath)
```

Comment:
unzip된 file들을 저장할 directory path로 추측되는데 맞다면
unzipFile -> unzipTargetDirectory, filePath -> unzipTargetDirectoryPath 로 변경한다면 directory임을 더 명확하게 인지할 수 있을 것 같아요
```suggestion
        val unzipTargetDirectory = File(unzipTargetDirectoryPath)
```

Reply:
수정 완료했습니다!